cmd/monitoring.go

```diff
@@ -0,0 +1,351 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/google/go-github/github"
```

Comment:
Please use `goimports` to format Go code.

Reply:
Done